templates/home/Features/features.html

```diff
@@ -3,6 +3,19 @@
 
 {% block content %}
 <br>
+
+<div class="heading text-center display-3" style="font-family: 'Syne Mono', monospace;">Features</div><br>
+
+<br>
+<h4 class="fea" style="margin-left:10px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   In-depth concept coverage</h4>
+<h4 class="fea" style="margin-right:86px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   One step solution</h4>
+<h4 class="fea" style="margin-right:68px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   24*7 Doubt solving</h4>
+<h4 class="fea" style="margin-right:90px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   1 on 1 Mentoring</h4>
+<h4 class="fea" style="margin-right:88px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   Project Guidance</h4>
+<h4 class="fea" style="margin-right:45px;"><img src="https://img.icons8.com/color/48/000000/idea.png" style="height:30px;"/>   Certification Included</h4>
+<br><br>
+
+{% endblock content %}
 <div class="heading text-center display-3">CodingEasy</div><br>
 
```

Comment:
You should delete lines 19 to 31. It should work fine then.

Reply:
Did it